ClientEntryLoadedListenerTest.load_listener_notified_but_add_listener_not_notified_after_loadAll test failure

`ClientEntryLoadedListenerTest.load_listener_notified_but_add_listener_not_notified_after_loadAll` seems to be flaky. Observed here: http://jenkins.hazelcast.com/job/Hazelcast-pr-builder/5079/testReport/junit/com.hazelcast.client.map/ClientEntryLoadedListenerTest/load_listener_notified_but_add_listener_not_notified_after_loadAll/

```
Error Message
expected:<5> but was:<3>
Stacktrace
java.lang.AssertionError: expected:<5> but was:<3>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.client.map.ClientEntryLoadedListenerTest$4.run(ClientEntryLoadedListenerTest.java:226)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1351)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1453)
	at com.hazelcast.client.map.ClientEntryLoadedListenerTest.load_listener_notified_but_add_listener_not_notified_after_loadAll(ClientEntryLoadedListenerTest.java:223)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/hazelcast/src/test/java/com/hazelcast/client/map/ClientEntryLoadedListenerTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/client/map/ClientEntryLoadedListenerTest.java
@@ -16,21 +16,21 @@
 
 package com.hazelcast.client.map;
 
+import com.hazelcast.client.test.ClientTestSupport;
 import com.hazelcast.client.test.TestHazelcastFactory;
 import com.hazelcast.config.Config;
 import com.hazelcast.config.MapStoreConfig;
 import com.hazelcast.core.EntryEvent;
 import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.EntryProcessor;
 import com.hazelcast.map.IMap;
 import com.hazelcast.map.MapLoader;
-import com.hazelcast.map.EntryProcessor;
 import com.hazelcast.map.listener.EntryAddedListener;
 import com.hazelcast.map.listener.EntryLoadedListener;
 import com.hazelcast.map.listener.EntryUpdatedListener;
 import com.hazelcast.map.listener.MapListener;
 import com.hazelcast.test.AssertTask;
 import com.hazelcast.test.HazelcastParallelClassRunner;
-import com.hazelcast.test.HazelcastTestSupport;
 import com.hazelcast.test.annotation.ParallelJVMTest;
 import com.hazelcast.test.annotation.QuickTest;
 import org.junit.AfterClass;
@@ -55,7 +55,7 @@ import static org.junit.Assert.assertEquals;
 
 @RunWith(HazelcastParallelClassRunner.class)
 @Category({QuickTest.class, ParallelJVMTest.class})
-public class ClientEntryLoadedListenerTest extends HazelcastTestSupport {
+public class ClientEntryLoadedListenerTest extends ClientTestSupport {
 
     private static final TestHazelcastFactory FACTORY = new TestHazelcastFactory();
 
@@ -210,6 +210,8 @@ public class ClientEntryLoadedListenerTest extends HazelcastTestSupport {
 
         IMap<Integer, Integer> map = client.getMap("load_listener_notified_but_add_listener_not_notified_after_loadAll");
         map.clear();
+
+        makeSureConnectedToServers(client, 2);
 
         MapListener listener = new LoadAndAddListener(loadEventCount, addEventCount);
         map.addEntryListener(listener, true);

